[EDU-4103] add unlink command docs (#818)
aziontech/docs: src/content/docs/en/pages/devtools/cli/azion-cli/commands/unlink/unlink.mdx

---
title: Azion CLI unlink

> description: Unlinking an existing project from Azion Edge platform.

Reviewer: ```suggestion
description: Unlink an existing project from Azion Edge Computing Platform.
```

meta_tags: cli, next, javascript, astro, vue, react, edge computing
namespace: documentation_cli_unlink
menu_namespace: cliMenuAlpha
permalink: /documentation/devtools/cli/unlink-command/
---

:::note 
In case one or more required fields aren't informed through the specific flags, an interactive message prompts and asks for the missing information.
:::

---

## Unlink

> The `azion unlink` command disconnects a local repo or project folder from Azion. The use of `--yes` option allows the action to complete without asking for confirmation. To get more information about this command, you can use the `-h` or `--help` option.

Reviewer: ```suggestion
The `azion unlink` command disconnects a local repo or project folder from Azion. The use of the `--yes` option allows the action to complete without asking for confirmation. To get more information about this command, you can use the `-h` or `--help` option.
```

## Usage

```bash
azion unlink
```

## Local Options

> ### help  

Reviewer: ```suggestion
### help

```

---
title: Azion CLI unlink
description: Unlink an existing project from Azion Edge Computing Platform.
meta_tags: cli, next, javascript, astro, vue, react, edge computing
namespace: documentation_cli_unlink
menu_namespace: cliMenuAlpha
permalink: /documentation/devtools/cli/unlink-command/
---

> :::note 
> In case one or more required fields aren't informed through the specific flags, an interactive message prompts and asks for the missing information.
> :::
> 
> ---
> 
> ## Unlink
> 
> The `azion unlink` command disconnects a local repo or project folder from Azion. The use of the `--yes` option allows the action to complete without asking for confirmation. To get more information about this command, you can use the `-h` or `--help` option.

Reviewer 2: ```suggestion
## Unlink

The `azion unlink` command disconnects a local repo or project folder from Azion. The use of the `--yes` option allows the action to complete without asking for confirmation. To get more information about this command, you can use the `-h` or `--help` option.

:::note 
In case one or more required fields aren't informed through the specific flags, an interactive message prompts and asks for the missing information.
:::
```

## Usage

```bash
azion unlink
```

> ## Local Options

Reviewer 3: ```suggestion
---

## Local Options
```
